### PR TITLE
Fixes #141

### LINKED ISSUE #141: describe --bless in the testing section
I don't think we talk about the new `--bless` option in the testing section, do we? You can pass it to `./x.py test` and it will update the UI reference files. This replaces the old script we used to run. 

We should also talk about rustfix.

### PR DESCRIPTION
**Pull request by oli-obk**
cc @nikomatsakis

### CODE REVIEW

**Comment by mark-i-m** [comment]
Actually, could you also add a bit on the run-rustfix header command and .fixed files?

**Comment by oli-obk** [comment]
I have never used that personally and I'm not sure if it already works correctly (for some definition of correctly)

cc @killercup

**Comment by mark-i-m** [comment]
Ping @oli-obk

File: src/tests/adding.md [hunk 1]
```diff
@@ -148,6 +148,9 @@ exhaustive. Header commands can generally be found by browsing the
 `TestProps` structure found in [`header.rs`] from the compiletest
 source.
 
+* `run-rustfix` for UI tests, indicates that the test produces
+  structured suggestions, which are then applied and the final
```
Comment (mark-i-m) [inline src/tests/adding.md]: Does it get compiled again? I think the output of rustfix is just compared against the .fixed file.

Comment (killercup) [reply src/tests/adding.md]: Yes, [it gets compiled again](https://github.com/rust-lang/rust/blob/b4463d788bfd30b622a87a0e6f8e9271b9102e50/src/tools/compiletest/src/runtest.rs#L2634-L2649)

Comment (mark-i-m) [reply src/tests/adding.md]: Ah, then @oli-obk, could you just add that the expected fixed code should be in .fixed? Then r=me

**Pull request merged**